src/devtools/client/debugger/src/reducers/pause.js

```diff
@@ -685,4 +692,23 @@ export function getPausePreviewLocation(state: State) {
   return state.pause.previewLocation;
 }
 
+export function getResumePoint(state, type) {
+  const thread = getCurrentThread(state);
+  const framePoints = getFramePositions(state, thread)?.positions.map(p => p.point);
+  const executionPoint = getThreadExecutionPoint(state, thread);
+
+  if (!framePoints || isTopFrame(state, thread)) {
+    return executionPoint;
+  }
+
+  if (type == "reverseStepOver" || type == "rewind") {
+    return findLast(framePoints, p => +p < +executionPoint);
```

Comment:
What is the rationale behind changing the origin of the step based on the kind of step being performed?  Also, this should use BigInt instead of + to coerce the point correctly.

Reply:
👍